runtime/near-vm-logic/tests/test_miscs.rs

```diff
@@ -183,7 +183,7 @@ fn test_log_utf16_number_limit() {
     assert_costs(map! {
         ExtCosts::base: max_number_logs + 1,
         ExtCosts::log_base: max_number_logs,
-        ExtCosts::log_byte: len * max_number_logs,
+        ExtCosts::log_byte: string.bytes().len() as u64 * max_number_logs,
```

Comment:
`string.len()` returns the number of bytes, so we don't need `.bytes()` unless you want to be super-explicit about the bytes.